src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow/integrationDirectoryApplicationRow.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import {Link} from 'react-router';
+import capitalize from 'lodash/capitalize';
+import omit from 'lodash/omit';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+import SentryTypes from 'app/sentryTypes';
+import {PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import CircleIndicator from 'app/components/circleIndicator';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import {Organization, SentryApp, SentryAppInstallation} from 'app/types';
+import theme from 'app/utils/theme';
+
+const INSTALLED = 'Installed';
+const NOT_INSTALLED = 'Not Installed';
+const PENDING = 'Pending';
```

Comment:
If these never overlap you could replace them with [an enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums).

If you want, you could also throw the values from the `colors` object in there too:
```js
enum Status {
  Installed: {
    text: "INSTALLED",
    color: "success"
  },
  ...
}
```

Reply:
@leeandher it seems that enums only support numeric and string-based enums, not objects. But I think that `const` provides adequate protection against being reassigned.

Reply:
cool! didn't know that, but if its clean and it works, were good.